### PR TITLE
updated lab to clarify instructions & make it a lil more generic

### PR DESCRIPTION
**Pull request by realAndrewCohn**
This lab can be assigned either before or after fetch so I wanted to make it a little more open ended. Also got rid of the opinionated advice about creating classes. Let the students figure that one out on their own

### CODE REVIEW

**Comment by realAndrewCohn** [comment]
Also, `json-server` will catch on fire if the data does not have an `id` attribute. I fixed that as well

File: README.md [hunk 1]
```diff
@@ -1,26 +1,49 @@
...
-We're building out a search feature in our application (no backend persistence). A user should be able to search for a Pokemon and flip that Pokemon card to see the back side.
+- We're building out a search feature in our application (no backend persistence).
+
+- A user should be able to search for a Pokemon and flip that Pokemon card to see its alternate sprite.
+
+- Two files containing the same information are included: `db.json` and `pokemon.js`. If you've learned `fetch`,
+  consider using `json-server` to spin up a simple RESTful API that will give you your pokemon data:
+  - `$ npm install -g json-server`
+  - `$ json-server --watch db.json`
+
```
Comment (aalexander3) [inline README.md:32]: We could consider adding how to import / require the data in pokemon.js if they have not yet used fetch.

**Review by aalexander3 — APPROVED** [review]
Looks good! Nothing blocking in my opinion. I like that its a little more dynamic and doesn't ask for classes - especially since we don't talk about them too much!

File: README.md (hunk 1 above)
Comment (realAndrewCohn) [reply README.md:32]: perf. Updated the README for students that do not know `fetch` yet

**Pull request merged**